### PR TITLE
Added Dropdown menu under Resources

### PR DESCRIPTION
**Pull request by kmonterr**
I added the For applicants, For awardees and For reviewers in the Resources Tab

Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/BRANCH_NAME/)

[Preview README for this branch](https://github.com/18F/nsf-sbir/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

/cc @relevant-people

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -18,6 +18,13 @@ primary:
     href: /showcase/
   - text: Resources
     href: /resources/
+      - text: For Applicants
+        href: /two/a/
+      - text: For Awardees
+        href: /two/b/
+      - text: For Reviewers
+        href: /two/c/
+
```
Comment (line47) [inline _data/navigation.yml]: The formatting is a bit off here. The `href: /resources/` should be removed and a `links:` header should be added. See the example below.

```
- text: Navigation section
    links:
      - text: Demo link A
        href: /two/a/
      - text: Demo link B
        href: /two/b/
      - text: Demo link C
        href: /two/c/
```

Comment (kmonterr) [inline _data/navigation.yml]: is that right?

Comment (line47) [inline _data/navigation.yml]: This is so close!  There are 2 text references and you only can have one. I would suggest removing the `text: Navigation section` and keep `- text: Resources`.

**Comment by line47** [comment]
@kmonterr This looks like it is working! https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/dropdown-in-resources/ 

Now If update the links, I can look into updating the styling

**Comment by kmonterr** [comment]
@line47 woohoo!! thank you! looks awesome! Should each resources page have the same styling?

**Comment by line47** [comment]
@kmonterr Yeah, I'm about to update some of the navigation styling for the dropdown 👍

**Pull request merged**